css/_chat.scss

```diff
@@ -214,12 +214,11 @@
 }
 
 .smiley {
-    height: 26px;
+    font-size: 14pt;
 }
 
 #smileys {
-    background: white;
-    border-radius: 50%;
+    font-size: 20pt;
```

Comment:
Why did you go with pts over px (or em)?

Reply:
Hum, no reason, probably I was looking at some other css values while trying to figure out the correct size. Should I change it?

Reply:
If you had not reason to use it then I do suggest you change it to use px or em. em scales relative to the parent, which is maybe what we want in the future (like 15 years from now, let's be honest). The easier would be px and that's the most common measurement used in the jitsi css.

Reply:
One thing I notice is that the whole _chat.scss has `pt` in it, it is not just this change.